### PR TITLE
Owner is now just symbollic

### PR DESCRIPTION
**Pull request by mredigonda**
This pull request will be deleted soon, I'll make these changes directly into Guillermo's pull request: #587

### CODE REVIEW

File: server/controllers/ticket/change-priority.php [hunk 1]
```diff
@@ -50,23 +50,18 @@ public function handler() {
...
+            'authorStaff' => Controller::getLoggedUser(),
+            'content' => $ticket->priority,
+            'date' => Date::getCurrentDate()
+        ));
+        $ticket->addEvent($event);
+        $ticket->store();
 
+        Log::createLog('PRIORITY_CHANGED', $ticket->ticketNumber);
+        Response::respondSuccess();
     }
```
Comment (ivandiazwm) [inline server/controllers/ticket/change-priority.php]: What about other paths like close/re-open/comment/change-department etc.?

Comment (mredigonda) [reply server/controllers/ticket/change-priority.php]: Close and comment were modified in this commit, change-department was working accordingly, and I'll check out about re-open.

**Comment by ivandiazwm** [comment]
Also take into account that the ticket must be from a department managed by the current staff to me viewed/edited. You'll need to use `canMangeTicket` https://github.com/opensupports/opensupports/pull/587/files#diff-e2b5f90b86cdb01606921c25c7180da3R52

File: server/controllers/ticket/change-priority.php (hunk 1 above)
Comment (ivandiazwm) [reply server/controllers/ticket/change-priority.php]: :v

Comment (ivandiazwm) [reply server/controllers/ticket/change-priority.php]: @guillegiu ^

**Comment by ivandiazwm** [comment]
@mredigonda I'm closing this because changes are included in the other PR

**Pull request closed without merging**